migrations_scripts/1_deploy_registry_and_add_adapters.js

```diff
@@ -287,6 +298,19 @@ module.exports = async (deployer, network, accounts) => {
     'protocol-icons.s3.amazonaws.com/curve.fi.png',
     '0',
   ]);
+  await deployer.deploy(DmmAssetAdapter, { from: accounts[0] })
+    .then(() => {
+      adapters[4].push(DmmAssetAdapter.address);
+      tokens[4].push(dmmAssetAdapterTokens);
+    });
+  protocolNames.push('DMM');
+  metadata.push([
+    'DMM',
```

Comment:
You should use full name here (and at line 306), as it will be more clear for users